WooCommercePOS/WooCommercePOS/Presentation/PlusButtonView.swift

```diff
@@ -0,0 +1,25 @@
+import SwiftUI
+
+struct PlusButtonView: View {
```

Comment:
nit: how about making it a style modifier like `PlusButtonStyle` in the main app?

Reply:
Sounds good. I've created a new "Reusable Views" folder with the PlusButtonStyle and adapted what we had for PlusIcon to out case: https://github.com/woocommerce/woocommerce-ios/pull/12720/commits/2bd611ce7f3fc76e86b8bf8a1293cb5fbeb89376